drivers/tty/serdev/pibridge.c

```diff
@@ -284,21 +301,32 @@ int pibridge_req_gate_tmt(u8 dst, u16 cmd, u8 *snd_buf, u8 snd_len,
 	}
 
 	if (to_discard) {
+		int ret;
```

Comment:
Huh? Looks like you already added another one further up which is scoped to the function?